### PR TITLE
Geocaching

### PR DESCRIPTION
**Pull request by Sholofly**
## Proposed change

Adds documentation for the new Geocaching integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [#50284](https://github.com/home-assistant/core/pull/50284)
- Link to parent pull request in the Brands repository: [#2555](https://github.com/home-assistant/brands/pull/2555)
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelated commits in this PR. Probably caused by basing this PR from the `current` branch (instead of the `next`). Please perform a `git rebase` onto the `next` branch to clean this up.

**Comment by Sholofly** [comment]
@frenck Check! Thanks for your help!

File: source/_integrations/geocaching.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ha_codeowners:
+  - '@Sholofly'
+  - '@reinder83'
+ha_domain: geocaching
+ha_platforms:
+  - sensor
+---
+
+The Geocaching integration can be used to monitor Geocaching account statistics.
+
```
Comment (klaasnicolaas) [inline source/_integrations/geocaching.markdown:22]:
```suggestion
{% include integrations/config_flow.md %}
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/geocaching.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - '@Sholofly'
+  - '@reinder83'
+ha_domain: geocaching
+ha_platforms:
+  - sensor
+---
+
+The Geocaching integration can be used to monitor Geocaching account statistics.
+{% include integrations/config_flow.md %}
+## Sensors
```
Comment (frenck) [inline source/_integrations/geocaching.markdown]: Adding a bit of body:

```suggestion
Geocaching is a real-world, outdoor adventure that is happening all the time, all around the world. To play, participants use the Geocaching app and/or a GPS device to navigate to cleverly hidden containers called geocaches.

The Geocaching integration in Home Assistant pulls data from your [Geocaching.com](https://www.geocaching.com/) account.

{% include integrations/config_flow.md %}

## Sensors
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by Sholofly** [comment]
@frenck & @klaasnicolaas, thanks!

**Comment by frenck** [comment]
Rebased onto the upstream `next` to fix the build issue.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
@Sholofly It seems like the source branch for this PR is missing, thus this PR cannot be adjusted anymore. 

Can you restore the source branch?

**Comment by Sholofly** [comment]
I've created a new PR (#22728 ) because I accidentally removed the original fork and wasn't able to restore it.

**Pull request closed without merging**